packages/rocketchat-livechat/client/views/app/tabbar/visitorNavigation.js

```diff
@@ -5,16 +5,16 @@ Template.visitorNavigation.helpers({
 		return !Template.instance().pageVisited.ready();
 	},
 
-	pageVisited() {
+	pages() {
 		const room = ChatRoom.findOne({ _id: this.rid }, { fields: { 'v.token': 1 } });
 
-		if (room && room.v && room.v.token) {
-			return LivechatPageVisited.find({ token: room.v.token }, { sort: { ts: -1 } });
+		if (room) {
+			return LivechatMessage.find({rid: room._id, t: 'livechat_navigation_history', $or: [{ _shared: { $exists: false }}, { _shared: true }]}, { sort: { ts: -1 } });
```

Comment:
why did you use this `_shared` filter?

Reply:
OMG! It's a good question...
As you could see, I made a mistake.. Actually, I was trying to deal with the `_hidden` property and not the `_shared` property, but I just see that it is not necessary because I'm filtering the messages type(`t: 'livechat_navigation_history'`) on the subscribe.
I made this mess because we have a setting to activate this feature and when the feature is not enabled, the messages are still sending, but with the `_hidden` property defined. So, I thought that if I didn't force the filter, the messages would not be received on the client side.
Anyway, I just pushed a new commit fixing this mistake.
Thanks!